general/administration/reverse-proxy.md

```diff
@@ -197,13 +198,8 @@ server {
 
     # Jellyfin
     location /jellyfin {
-        return 302 $scheme://$host/jellyfin/;
-    }
-
-    location /jellyfin/ {
         # Proxy main Jellyfin traffic
-        # The / at the end is significant.
-        proxy_pass http://SERVER_IP_ADDRESS:8096/;
+        proxy_pass http://SERVER_IP_ADDRESS:8096;
```

Comment:
Do we want to remove the trailing slash?

https://github.com/GrrrDog/weird_proxies/wiki/nginx#proxy_pass

Reply:
Not if we're using the base url.  A trailing slash more or less strips part of the passed URL string out when passing it to the backend.  Can't check right now, but if I remember right then it strips out the location block it's in.  So
```
location /jellyfin {
    proxy_pass http://$IP:8096/
```
would only pass `http://$IP:8096/web/index.html` back.

If we have a baseurl of `/jellyfin` set, then we want to make sure that the full `http://domain.com/jellyfin/whatever` gets passed back

Reply:
So if the internal service as a Path, I know you can handle it with traefik. I imagine nginx also does too (since nginx is ya kno, better lol).

Does it fail to parse correctly when jellyfin doesn't have a base url set? Or is the lack of closing slash only needed when a base url is set?

Reply:
What's in the repo now should be pretty close to working for no base url but running JF in a sub-path (I admittedly didn't check), but it presents other problems.  For example, Jellyfin then isn't aware that it's running behind a sub-path so things like autodiscovery won't work.  Jellyfin will announce "hey I'm at http://192.168.1.10" instead of `http://192.168.1.10/jellyfin`.  If we're going to provide documentation to use a sub-path, we should probably recommend the base url approach to have a more complete solution.

Reply:
What I mean is with the server at http://192.168.1.10/jellyfin can you assign the proxy pass to appropriately pass the entire line include the Path variable. I've had to do this with pihole to normalize the FQDN url

I think this is sort of what I mean? 
https://serverfault.com/questions/527907/redirect-a-subpath-to-a-external-host-with-nginx